src/Middleware/NavigationMiddlewareFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Zend\Navigation\Middleware;
+
+use Interop\Container\ContainerInterface;
+use Zend\Navigation\Navigation;
+
+class NavigationMiddlewareFactory
+{
+    /**
+     * Top-level configuration key indicating navigation configuration
+     *
+     * @var string
```

Comment:
Ok fine. Please just note it is proposed standard, not yet accepted. It's a bit odd that _constant_ has annotation _variable_, isn't it?